src/Admin.php

```diff
@@ -277,4 +281,26 @@ public static function wp_dropdown_users_args(array $query_args) {
     return $query_args;
   }
 
+  /**
+   * @implements wp_upgrade
+   */
+  public static function wp_upgrade($wp_db_version, $wp_current_db_version) {
+    static::createUploadsHtaccessFile();
+  }
+
+  /**
+   * @implements wp_install
+   */
+  public static function wp_install($wp_user) {
+    static::createUploadsHtaccessFile();
+  }
+
+  /**
+   * Copies .htaccess template file into uploads directory.
+   */
+  public static function createUploadsHtaccessFile() {
+    $upload_dir = explode('sites/', wp_upload_dir(NULL, FALSE)['basedir'])[0];
+    copy(dirname(__DIR__) . '/conf/.htaccess.uploads', $upload_dir . '/.htaccess');
```

Comment:
1. Before overwriting an existing file, the code should check whether the existing file contains the content of the template file already.  Doesn't have to be identical, there may be additional lines above or below, but the template should be fully contained.  The method name should be adjusted accordingly from "create" into "ensure".

1. You also need to check whether the target file is writable.

1. A similar operation should be performed on the .gitignore file.

1. When operating on WP-CLI, there should be corresponding success or error messages, so that the user is aware that additional operations were (attempted to be) performed.